package.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Think about dev and prod dependencies

Reply:
I thought about them a lot))
Checked the install process for most of them here: https://www.npmjs.com/

Reply:
I have also read this Webpack 4 Tutorial (https://www.valentinog.com/blog/webpack/) and watched how this guy handles dependencies.
But he even managed to put Vue in devDependencies)), which is questionable, since we supposedely will need Vue at runtime.

Reply:
Made some changes to dependencies. Please, have a look when you can, if it's ok now or there's still something that I should change.